backend/src/main/resources/application.yaml

```diff
@@ -26,6 +26,10 @@ spring:
     properties:
       hibernate:
         default_schema: public # Using the public schema for now, so we don't have to add a manual step to the deploy process
+        # generate_statistics: true # for observing batching
+        jdbc:
+          batch_size: 100
+        order_inserts: true
```

Comment:
@georgehager do you know whats the default setting for this feature? I am a bit concerned about any potential side effects

Reply:
I would recommend implementing a naive chunking solution on the service side since that will address the batching on the jdbc side and also address the potential OOM error because the Set size, rather than turning this setting on for everything.

Reply:
After a bit of reading, my opinion is we should do the naive chunking as you suggest in batches of 1000, which would pass at most 1000 records to `saveAll`, _and_ we should turn on `jdbc.batch_size` as well, without `order_inserts`. This would give us good memory safety and improved performance. I don't believe we should see side effects from just turning on `jdbc.batch_size` but I'll do some testing. @emmastephenson

Reply:
Long-term, I agree with @georgehager that including both naive chunking and batching the JDBC saves will give us the best performance. In the interest of keeping this PR small, however, I've created a new ticket to address the JDBC batching: https://github.com/CDCgov/prime-simplereport/issues/4864